include/mvdtool/bits/sonata_misc.hpp

```diff
@@ -20,18 +20,18 @@
 
 #include <string>
 #include <vector>
-#include <unordered_set>
 
 #include <boost/math/quaternion.hpp>
 
 #include <highfive/H5DataSet.hpp>
 
 #include "../sonata.hpp"
```

Comment:
Should we remove this include?

Reply:
? Im puzzled. The change was to remove, we are not using it, right?

Reply:
I'm talk about `../sonata.pp`

Reply:
Oh... humm.. it's part of the convention here... `mvd3_misc` also includes `mvd3.hpp`... Shall we remove in both places? But it doesnt hurt, no?

Reply:
Yes, I tried building the project without `#include "../sonata.hpp"` and `#include "../mvd3.hpp"` in the corresponding files and it built without any issue. I think that since it is redundant it can be removed. I also don't like that in `mvd3.hpp`, `sonata.hpp` and `tsv.hpp` the `#include "bits/<type>.hpp"` is in the end of the file but all those are things that we can take care in seperate PR, maybe together with the changes that `make clang-format` will introduce.

Reply:
include bits at the end of files is quite common

Reply:
Ah okay then, I wasn't familiar with this :+1:

Reply:
That's header only stuff